verify.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once 'functions.php';
+if((!isset($_POST['charname']) || $_POST['charname'] === '') 
+    || (!isset($_POST['class']) || $_POST['class'] === '')
+    || (!isset($_POST['level']) || $_POST['level'] === '')
+    || (!isset($_POST['strength']) || $_POST['strength'] === '')
+    || (!isset($_POST['dexterity']) || $_POST['dexterity'] === '')
+    || (!isset($_POST['constitution']) || $_POST['constitution'] === '')
+    || (!isset($_POST['intelligence']) || $_POST['intelligence'] === '') 
+    || (!isset($_POST['wisdom']) || $_POST['wisdom'] === '') 
+    || (!isset($_POST['charisma']) || $_POST['charisma'] === '')) {
+        header('location: char_creation.php?error=All fields needed');
+} elseif ($_POST['strength'] > 20 
+    || $_POST['dexterity'] > 20 
+    || $_POST['constitution'] > 20 
+    || $_POST['intelligence'] > 20 
+    || $_POST['wisdom'] > 20 
+    || $_POST['charisma'] > 20) {
```

Comment:
Same deal with this slightly less chunky conditional. If you make it into some kind of validateFields (or something like that) function you could probably re-use it in other parts of the application.